CONTRIBUTING.md

```diff
@@ -221,7 +221,7 @@ $ ./build.sh dask_cudf
 - To run Python tests (Optional):
 ```bash
 $ cd $CUDF_HOME/python
-$ py.test -v cudf                           # run cudf test suite
+$ py.test -v cudf/cudf/tests                # run cudf test suite
 $ py.test -v dask_cudf                      # run dask_cudf test suite
```

Comment:
Do we need something similar for `dask_cudf`?

Reply:
No it doesn't need it. Please I'd like to know what does 'XFAILED' mean for a unit test case instead of FAILED.

Reply:
From the pytest docs,
```
An xfail means that you expect a test to fail for some reason. A common example is a test for a feature not yet 
implemented, or a bug not yet fixed. When a test passes despite being expected to fail (marked with pytest.mark.xfail), it’s 
an xpass and will be reported in the test summary.
```
https://docs.pytest.org/en/latest/how-to/skipping.html

Reply:
Thank you @kshitij12345, I thought it was a `cudf` specific feature.